### DISCUSSION TITLE
QT6?

### DISCUSSION THREAD

**Discussion by Podberezovik34** [smplayer-dev/smplayer#1058] [General]
Hello! Are you planning to transfer the application to support QT 6?

**Comment by smplayer-dev** [comment]
I guess it would have to be done eventually.

**Reply by Podberezovik34** [reply]
May I ask, in this decade?

**Reply by smplayer-dev** [reply]
It's already done:
https://github.com/smplayer-dev/smplayer/tree/qt6

**Reply by Podberezovik34** [reply]
In which release version? No matter which version I try to install, qt5 pulls in dependencies.

**Reply by smplayer-dev** [reply]
There isn't any release yet with Qt 6. Only this package for windows for testing:
https://github.com/smplayer-dev/smplayer/actions/runs/17513441959/artifacts/3943587493
